# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/rsbac/rsbac-no_extraversion.diff
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	Fri Mar  7 16:24:43 2003
+++ ./Makefile	Fri Mar  7 16:25:10 2003
@@ -71,9 +71,9 @@
 #
 # Add RSBAC version
 #
-ifeq ($(CONFIG_RSBAC),y)
-EXTRAVERSION=-rsbac
-endif
+#ifeq ($(CONFIG_RSBAC),y)
+#EXTRAVERSION=-rsbac
+#endif
 
